src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
+* {
+  display: flex;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+}
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  background-color: #d4e2e6;
+  border: 1vmin #000 dotted;
+  border-radius: 50%;
+  position: relative;
+
+  &-minute,
+  &-second {
+    position: absolute;
+    transform-origin: bottom;
+    bottom: 50%;
```

Comment:
it works and you don't duplicate styles, but you write it separately. for such cases is good to create a common class with styles, like `stopwatch__hand`. In HTML add this class to both - second and minute hands